organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+The Mission of Hacker Society
+=============================
+
+A time and place for people to learn about technology from each other, in order
+to build a community, to empower the individual, promote equality, and build an
```

Comment:
you're goin' cray with your infinitives here. 
"to build a community, to empower the individual, TO promote equality, and TO build an open society"

Also, you're using "build" twice. Replace one with a fun synonym like "foster" or "create!"